target/linux/ramips/dts/mt7621_iodata_wn-dx1200gr.dts

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621_iodata_wn-xx-xr.dtsi"
+
+/ {
+	compatible = "iodata,wn-dx1200gr", "mediatek,mt7621-soc";
+	model = "I-O DATA WN-DX1200GR";
+};
+
+&leds {
+	wps {
+		label = "blue:wps";
+		gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+	};
+};
+
+&factory {
+	reg = <0x0200000 0x0200000>;
+};
+
+&partitions {
+	/delete-node/ partition@300000;
```

Comment:
Stuff like this and the factory node above IMO make this really hard to read.
In this case I think I prefer a full DTS file without DTSI.

Reply:
OK, I also think so. I'll update.